### ISSUE TITLE
upload this package to bintray

### ISSUE THREAD

**Issue by gocarlos** [steakhal/conan-prometheus-cpp#2]
I do not seem to find this package on bintray. 

it would be great if you could upload it and adjust the usage instructions to the readme

**Comment by steakhal** [comment]
You are right. The only problem is that this package is not ready yet. I have some linkage issues which should be addressed before the upload.

If you can help me out with them than we could make some progress.
Currently I don't have much time to fix this :(

The only reason that this repo is public that maybe some would join and finish it :D
